frontend/pages/index.tsx

```diff
@@ -1,11 +1,93 @@
-import { Center } from '@mantine/core';
+import { Anchor, Center, Divider, List, Text, Title } from '@mantine/core';
 
 export default function Home() {
   return (
-    <>
-      <Center style={{ alignItems: 'center', paddingTop: '80px' }}>
-        <div>Coming soon!</div>
+    <div style={{ display: 'flex', height: '100%', width: '100%', flexDirection: 'column', padding: '4px 20px' }}>
+      <Center style={{ flexDirection: 'column', width: '100%' }}>
+        <Title order={1} underline>
+          Measure Repository Service
+        </Title>
       </Center>
-    </>
+      <Title order={2} underline>
+        Summary:
+      </Title>
+      <Text>
+        This application is an interface for a prototype implementation of a{' '}
+        <Anchor href="https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html">
+          FHIR Measure Repository Service
+        </Anchor>{' '}
+        with Measure and Library authoring capabilities.
+      </Text>
+      <Divider my="sm" variant="dotted" />
+      <Title order={2} underline>
+        Getting Started:
+      </Title>
+      <Text>
+        The backend Measure Repository Service server must be running on port 3000 in order for this application to pull
+        data from the server. To start the backend Measure Repository Service server on port 3000, run &quot;npm run
+        start:all&quot; or &quot;npm run start:backend&quot; from the root of the repository.
+      </Text>
+      <Text>More functionality coming soon!</Text>
+      <Divider my="sm" variant="dotted" />
+      <Title order={2} underline style={{ marginBottom: '8px' }}>
+        Capabilities
+      </Title>
+      <Title order={3} underline>
+        Resource Read and Search
+      </Title>
+      <Text style={{ marginBottom: '8px' }}>
+        This application provides and interface for Measure Repository Service read and search operations. Users can
+        view the JSON content of resources on the server by clicking the desired resource resource type in the
```

Comment:
```suggestion
        view the JSON content of resources on the server by clicking the desired resource type in the
```
One too many resources?